### ISSUE TITLE
IoT meetup --- what do you want to learn / what do you want to teach

### ISSUE THREAD

**Issue by Yaakov-Belch** [Yaakov-Belch/mqtt-meetup#6]
The majority of us has some interest in IoT.  So --- let's organize a meetup on IoT.

But IoT is a big field.  Can everybody say a few words what he is interested in --- to learn or to teach --- or just to discuss?

**Comment by Yaakov-Belch** [comment]
A simple IoT demo application.

I have have bought:

* a simple arduino with built-in WiFi,
* a LCD shield with six push buttons,
* a buzzer.

I want to wire them up and program the arduino so that it connects through WiFi to a MQTT broker on my Linux server to:

* publishes button press events,
* subscribe to text events to display on the LCD display
* subscribe to events to switch the buzzer on or off.

In other words --- start a very basic arduino project.  I guess I need just a few URLs to follow.
